content/sensu-go/6.9/release-notes.md

```diff
@@ -113,6 +114,17 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.9.2 release notes
+
+**March 8, 2023** &mdash; The latest release of Sensu Go, version 6.9.2, is now available for download.
+
+Sensu Go 6.9.2 fixes a UI crash on the Service Component detail page. Additionally we added a GraphQL validator for query node depth.
+
+**FIXES:**
+
+- ([Commercial feature][303]) In the web UI, added GraphQL validator for query node depth.
```

Comment:
This is the XSS issue, isn't it? We should probably be a bit more descriptive about what we're actually fixing.